Makefile.common

```diff
@@ -115,7 +115,7 @@ common-test:
 .PHONY: common-format
 common-format:
 	@echo ">> formatting code"
-	GO111MODULE=$(GO111MODULE) $(GO) fmt $(GOOPTS) $(pkgs)
+	GO111MODULE=$(GO111MODULE) $(GO) fmt $(pkgs)
```

Comment:
Question : Can we use `goimports` instead? Since `goimports` does what `go fmt` does and more.

Reply:
The problem with `goimports` is that it doesn't flag ambiguities. It just picks one of the possible imports if there are multiple possibilities to fix a missing import. Thus, its usage is not entirely uncontroversial.

Reply:
Got it! Thanks for the clarification :)